Documentation/ceph-common-issues.md

```diff
@@ -988,44 +976,32 @@ You can see https://github.com/rook/rook/issues/7940 for more detailed informati
 
 ### Solution
 #### Recover from corruption (v1.6.0-v1.6.6)
-If you are using Rook v1.6, you must update to v1.6.7 or higher to avoid further incidents of OSD
-corruption caused by these Atari partitions.
-
-If there are already OSDs created on the unexpected partitions, OSD corruption has already occurred.
-To recover from this, immediately update to v1.6.7 or higher. After the update, no further OSD
-corruption should occur. Next, to get back to a healthy Ceph cluster state, focus on one corruped
-disk at a time and [remove all OSDs on each corrupted disk](ceph-osd-mgmt.md#remove-an-osd) one disk
-at a time.
-
-As an example, you may have `/dev/sdb` with two unexpected partitions (`/dev/sdb1` and `/dev/sdb2`)
-as well as a second corrupted disk `/dev/sde` with one unexpected partition (`/dev/sde1`).
-1. First, remove the 3 OSDs associated with `/dev/sdb`, `/dev/sdb1`, and `/dev/sdb2`.
-2. Then wipe clean `/dev/sdb`.
-3. After this, allow the Rook operator to deploy a new OSD to `/dev/sdb`; this will allow Ceph to
+If you are using Rook v1.6, you must first update to v1.6.8 or higher to avoid further incidents of
+OSD corruption caused by these Atari partitions.
+
+An old workaround suggested using `deviceFilter: ^sd[a-z]+$`, but this still results in unexpected
+partitions. Rook will merely stop creating new OSDs on the partitions. It does not fix a related
+issue that `ceph-volume` that is unaware of the Atari partition problem. Users who used this
+workaround are still at risk for OSD failures in the future.
```

Comment:
What is the risk of failure besides an extra partition showing up? The corruption will only come if the additional OSD is created on the atari partition, right?

Reply:
No. See the explanation here: https://github.com/rook/rook/issues/7940#issuecomment-880958164